sys/zuse.hoon

```diff
@@ -3348,21 +3348,26 @@
       ==
     ::
     ++  secp
-      |=  [w=@ p=@ a=@ b=@ g=pont n=@]
+     ~/  %secp
+      |=  [w=@ p=@ a=@ b=@ g=pont n=@]  :: being passed in from above
       =/  p  ~(. fo p)
       =/  n  ~(. fo n)
+      ~%  %helper  ..$
```

Comment:
Syntax error on the line below, I think you're missing a `~` here? ie `~%  %helper  ..$  ~`

Reply:
on it.

Reply:
and, 1 month after doing it, actually pushing it...